sass/_header.scss

```diff
@@ -479,14 +479,14 @@ li.search {
       @include input;
       box-shadow: inset 0 0 0 2px $nypl-search-color-dark;
       height: 2rem;
-      margin-right: 1px;
+      margin: 0.3125rem 0.3125rem 0;
```

Comment:
this produces a border around the search box: 
![image](https://cloud.githubusercontent.com/assets/133020/21654246/b4def3e0-d282-11e6-95bd-2ad76f6066f4.png)